examples/example.tf

```diff
@@ -3,7 +3,7 @@ resource "virtualbox_vm" "node" {
     count = 2
     name = "${format("node-%02d", count.index+1)}"
     url = "https://vagrantcloud.com/ubuntu/boxes/trusty64/versions/20180206.0.0/providers/virtualbox.box"
-    image = "./virtualbox-ubuntu.box"
+    image = "./virtualbox.box"
```

Comment:
This change would not be needed after #40

Reply:
@VoyTechnology since #40 is merged, I guess the indicated change should be reverted?